tasks.py

```diff
@@ -170,43 +246,63 @@ def black(context, autoformat=False):
         black_command = "black"
     else:
         black_command = "black --check --diff"
-    docker_compose(
-        context,
-        f"run --entrypoint '{black_command} development/ nautobot/ tasks.py' nautobot",
-        pty=True,
-    )
+
+    command = f"{black_command} development/ nautobot/ tasks.py"
+
+    run_command(context, command)
 
 
 @task
 def flake8(context):
     """Check for PEP8 compliance and other style issues."""
-    docker_compose(context, "run --entrypoint 'flake8 development/ nautobot/ tasks.py' nautobot", pty=True)
+    command = "flake8 development/ nautobot/ tasks.py"
+
+    run_command(context, command)
+
+
+@task
+def check_migrations(context):
+    """Check for missing migrations."""
+    command = "nautobot-server --config=nautobot/core/tests/nautobot_config.py makemigrations --dry-run --check"
+
+    run_command(context, command)
 
 
 @task(
     help={
         "keepdb": "save and re-use test database between test runs for faster re-testing.",
         "label": "specify a directory or module to test instead of running all Nautobot tests",
+        "failfast": "fail as soon as a single test fails don't run the entire test suite",
     }
 )
-def unittest(context, keepdb=False, label="nautobot"):
+def unittest(context, keepdb=False, label="nautobot", failfast=False):
```

Comment:
Can you please add support for `nautobot-server test -b/--buffer` here:

Silences output from passing tests (makes tests less noisy, this should be the default IMO), so `buffer=True`?
```
  -b, --buffer          Discard output from passing tests
```